awesomerc.lua

```diff
@@ -13,6 +13,8 @@ local wibox = require("wibox")
 local beautiful = require("beautiful")
 -- Notification library
 local naughty = require("naughty")
+-- Declarative object management
+local ruled = require("ruled")
```

Comment:
I re-ordered the commits. The CI doesn't detect it because `awesomerc.lua` is post-processed by CMake. Another instance of this causing CI issues is when a file is added in a later commit, "check each commit" will always have it since its copy will be in the build directory. This happens because Travis run make for the last commit in the PR. This calls `configure_file`. Then the first commit is checked out. So the build directory is polluted with artifacts of the last commit.